### ISSUE TITLE
Graph Text Color

### ISSUE THREAD

**Issue by ObaoziO** [RITAccess/Glance-data-analyzer#140]
Graph text color doesn't reset back to black/original text color when uncheck contrast.

**Comment by jmiller656** [comment]
This is how it is supposed to work. We want to guarantee that the graph labels are readable, so we do not include the choice for them to not contrast with the background.

**Comment by baozi90** [comment]
So the thing is, when without contract and the graph background is changed to black, the text will stay black. If the contrast in checked the color with change. 

By unchecking the check box I was thinking that the text might go back to black/the original color. Otherwise, there isn't a point to uncheck the contrast since it'd stay the contrast color after it's enabled.

**Comment by baozi90** [comment]
A better example might be a red background color with original/black text. I check the contrast and the text color turned into white, but I want the original black text color so I would uncheck the contrast, but then nothing happens.

**Comment by baozi90** [comment]
At first, unchecking the contrast will return the graph text color to original. (After the second change of background color, the text color stays contrasted even if the contrast is disabled.) >> At least that's what I thought at first, but it seems it just changes text color somehow when the background is changed even though the contrast is disabled. 

For example, background is black and contrast disabled, the text is black. Enable contrast, text turns white. Disable contrast again, text turns black (original color). Change background to red as contrast stays disabled, text turns white, which should stay black (original color) instead. 

To summarize, when contrast is enabled once, (even when background wasn't changed at first), then disabled, the text color changes while the background color changes. Text color doesn't always change with the change in background color, and it doesn't seem to change by contrast of background color either.

**Comment by ObaoziO** [comment]
Is the last commit supposed to make it so the graph background and text color are contrasted even though the "High Contrast" is not checked? On Safari, Chrome, and Firefox, that's what is happening so far.

**Comment by jmiller656** [comment]
Yeah, it seemed counterintuitive to allow the labels to be unreadable.

**Comment by ObaoziO** [comment]
Then I guess the High Contrast should be changed to checked at all times and can't be unchecked by user.

**Comment by ObaoziO** [comment]
I've disabled the checkbox and set it as checked. Also, the related codes for graphConstrast, in the reset.js, is commented out for now.
